packages/modeling/src/color/color.js

```diff
@@ -1,10 +1,37 @@
 const flatten = require('../utils/flatten')
 
+const geom2 = require('../geometry/geom2')
+const geom3 = require('../geometry/geom3')
+const path2 = require('../geometry/path2')
+const poly3 = require('../geometry/poly3')
+
+const colorGeom2 = (color, object) => {
+  const newgeom2 = geom2.create(geom2.toSides(object))
+  newgeom2.color = color
+  return newgeom2
+}
+
+const colorGeom3 = (color, object) => {
```

Comment:
Imho, I think each of these belongs inside their specific geometries folder (ie geom2/color, geom3/color) but that is a change that can be done later, that I would love to tackle :)